app/src/main/java/org/mozilla/focus/telemetry/startuptelemetry/StartupTypeTelemetry.kt

```diff
@@ -69,8 +73,11 @@ class StartupTypeTelemetry(
         val startupPath = startupPathProvider.startupPathForActivity
         val label = getTelemetryLabel(startupState, startupPath)
 
-        PerfStartup.startupType[label].add(1)
-        logger.info("Recorded start up: $label")
+        @OptIn(DelicateCoroutinesApi::class)
+        GlobalScope.launch(Dispatchers.IO) {
+            PerfStartup.startupType[label].add(1)
+            logger.info("Recorded start up: $label")
+        }
```

Comment:
This may cause tests to fail since it moves execution to another thread while tests run on Main.
Similar to https://github.com/mozilla-mobile/fenix/pull/26544/files we'd need a way to have the threading controllable from tests. 
@badboy We can help with this to get the tests to pass in CI and then land this patch.